parser/svc.go

```diff
@@ -91,7 +94,58 @@ PkgLoop:
 	}
 }
 
-// parseFuncs parses the pkg for any declared RPCs and auth handlers.
+// parseServiceStruct parses the pkg for any declared encore:service structs.
+func (p *parser) parseServiceStruct(pkg *est.Package, svc *est.Service) *est.ServiceStruct {
```

Comment:
The errors on this file don't always seem super useful to users. If possible could we do a second pass through them all and add links to the DI docs page (like we did for Cron's and PubSub)